components/create-collective/NewCreateCollective.js

```diff
@@ -0,0 +1,223 @@
+import React, { Fragment, Component } from 'react';
+import PropTypes from 'prop-types';
+import { Flex, Box } from '@rebass/grid';
+import { get } from 'lodash';
+import { defineMessages, injectIntl } from 'react-intl';
+
+import Page from '../Page';
+import { H1, P } from '../Text';
+import CreateCollectiveForm from './sections/CreateCollectiveForm';
+import CollectiveCategoryPicker from './sections/CollectiveCategoryPicker';
+import ConnectGithub from './sections/ConnectGithub';
+import SignInOrJoinFree from '../SignInOrJoinFree';
+import { withUser } from '../UserProvider';
+
+import { addCreateCollectiveMutationV2 } from '../../lib/graphql/mutations';
```

Comment:
A new pattern is to just have the mutation here in the component, to make clear it's only used there.

Reply:
Ok, I will move it.